pkg/templates/output.go

```diff
@@ -0,0 +1,53 @@
+package templates
+
+import (
+	"bufio"
+	"html/template"
+	"os"
+	"os/exec"
+
+	"github.com/fogo-sh/almanac/pkg/content"
+)
+
+func OutputAllPagesToDisk(pages map[string]*content.Page, outputDir string) error {
+	os.RemoveAll(outputDir)
+
+	err := os.MkdirAll(outputDir, 0755)
+	if err != nil {
+		return err
+	}
+
+	allPageTitles := content.AllPageTitles(pages)
+
+	for key, page := range pages {
+		var outputPath = outputDir + key + ".html"
+
+		f, err := os.Create(outputPath)
+		if err != nil {
+			return err
+		}
+
+		defer f.Close()
+
+		w := bufio.NewWriter(f)
+
+		err = PageTemplate.Execute(w, PageTemplateData{
+			AllPageTitles: allPageTitles,
+			Title:         page.Title,
+			Content:       template.HTML(string(page.ParsedContent)),
+			Backlinks:     page.Backlinks,
+		})
+		if err != nil {
+			return err
+		}
+
+		err = w.Flush()
+		if err != nil {
+			return err
+		}
+	}
+
+	exec.Command("cp", "-r", "pkg/static/static/.", outputDir).Run()
```

Comment:
@nint8835 , if there is a nice go util you know of to copy without calling out to cp, let me know :)

i am not happy with this but it does get the job done

Reply:
This might be worth a shot. Hate that this is something missing from the stdlib

https://github.com/otiai10/copy